Lucerne/MsEth1.xml

```diff
@@ -33,12 +33,13 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                   <idno facs="Luzern/01/" n="12">Ms Eth 01 (placeholder shelfmark)</idno>
                </msIdentifier>
                <msContents>
-                  <summary>Leporello with prayers፡</summary>
+                  <summary>Leporello with prayers</summary>
                   <msItem xml:id="ms_i1">
-                     <title>Protective prayer</title>
-                  </msItem>
-                  <msItem xml:id="ms_i2">
-                     <title>Protective prayer</title>
+                     <title ref="LIT5888AsmatPrayer">Protective prayer</title>
```

Comment:
It is better to identify and if necessary create new work IDs for protective prayers. Experience has shown that they can be often identified. The ID LIT5888AsmatPrayer was (if my memory is correct...) created for quick cataloguing in EMIP. It is not wrong, but if there is time, more specification would be beneficial.